README.rst

```diff
@@ -71,6 +71,7 @@ Docker
 
 Run subliminal in a docker container::
 
+    $ docker pull ghcr.io/diaoul/subliminal
     $ docker run --rm --name subliminal -v subliminal_cache:/usr/src/cache -v /tvshows:/tvshows -it diaoulael/subliminal download -l en /tvshows/The.Big.Bang.Theory.S05E18.HDTV.x264-LOL.mp4
```

Comment:
incorrect, if you did `pull` on `ghcr.io/diaoul/subliminal` you should also use it in `run` (you have `diaoulael/subliminal` for `run`)

Reply:
Seems to be addressed by this pr:
- https://github.com/Diaoul/subliminal/pull/1257

@getzze backreference (reply in comments) always good. makes people happy that their report was heard, and others who follow discussion also can understand.